uaclient/entitlements/livepatch.py

```diff
@@ -100,9 +102,17 @@ def enable(self, *, silent_if_inapplicable: bool = False) -> bool:
                     "/usr/bin/snap is present but snapd is not installed;"
                     " cannot enable {}".format(self.title)
                 )
-            util.subp(
-                [SNAP_CMD, "wait", "system", "seed.loaded"], capture=True
-            )
+
+            try:
+                util.subp(
+                    [SNAP_CMD, "wait", "system", "seed.loaded"], capture=True
+                )
+            except util.ProcessExecutionError as e:
+                if re.search(r"unknown command .*wait", str(e).lower()):
+                    logging.warning(
+                        status.MESSAGE_SNAPD_DOES_NOT_HAVE_WAIT_CMD
+                    )
+
```

Comment:
@lucasmoura I think we need a raise here right to avoid silently succeeding for any unmatched ProcessExecutionErrors 
```suggestion
                if re.search(r"unknown command .*wait", str(e).lower()):
                    logging.warning(
                        status.MESSAGE_SNAPD_DOES_NOT_HAVE_WAIT_CMD
                    )
                else:
                    raise
```